spec/javascripts/app/collections/aspect_selections_spec.js

```diff
@@ -0,0 +1,87 @@
+describe("app.collections.AspectSelections", function() {
+  beforeEach(function() {
+    var myAspects = [
+      {name: "Work", selected: true},
+      {name: "Friends", selected: false},
+      {name: "Acquaintances", selected: false}
+    ];
+    this.aspects = new app.collections.AspectSelections(myAspects);
+  });
+
+  describe("#selectAll", function() {
+    it("selects every aspect in the collection", function() {
+      this.aspects.selectAll();
+      this.aspects.each(function(aspect) {
+        expect(aspect.get("selected")).toBeTruthy();
+      });
+    });
+  });
+
+  describe("#deselectAll", function() {
+    it("deselects every aspect in the collection", function() {
+      this.aspects.deselectAll();
+      this.aspects.each(function(aspect) {
+        expect(aspect.get("selected")).toBeFalsy();
+      });
+    });
+  });
+
+  describe("#allSelected", function() {
+    it("returns true if every aspect is selected", function() {
+      this.aspects.at(1).set("selected", true);
+      this.aspects.at(2).set("selected", true);
+      expect(this.aspects.allSelected()).toBeTruthy();
+    });
+
+    it("returns false if at least one aspect is not selected", function() {
+      expect(this.aspects.allSelected()).toBeFalsy();
+    });
+  });
+
+  describe("#toSentence", function() {
+    describe("without aspects", function() {
```

Comment:
You also have an aspect here but it's not selected. Maybe merge this with `describe("with one aspect", ...` below and just select the aspect in the second test.